status-im/status-react
# Custom fonts loading added to desktop app

fixes #4860

### Summary:

- fonts for macos added to resources
- desktop files updated to load fonts from resources (changes copied from already existing feature in `react-native-desktop`
- references to new fonts added in `status-im.desktop.platform.cljs`

### Testing notes (optional):
Text input fields like chat adding and new contact adding doesn't support fonts changing due to react-native issue. Will be fixed automatically when this PR merged - https://github.com/status-im/react-native-desktop/pull/313https://github.com/status-im/react-native-desktop/pull/313

### Steps to test:
- Open Status
- Navigate to different pages, make sure that San Francisco font used on mac, or Arial on linux


status: ready
